cogs/tmpc.py

```diff
@@ -565,11 +527,11 @@ async def leave(self, ctx: Context):
             await document.chat.edit(overwrites=chat_overwrites)
 
     async def check_tmpc_channel(self, member: Member, _id: int, is_mod: bool = False, everyone: bool = False) \
-            -> Union[GamingChannel, StudyChannel]:
+            -> TempChannel:
         key = DBKeyWrapperEnum.CHAT.value
-        document: Union[GamingChannel, StudyChannel] = await self.study_db.find_one({key: _id})
+        document: TempChannel = await self.channel_db.find_one({key: _id})
         if not document:
-            document = await self.gaming_db.find_one({key: _id})
+            document = await self.channel_db.find_one({key: _id})
```

Comment:
not needed

Reply:
Request to change?

Reply:
yeah the `if not document` is always false

Reply:
Done, please wait for push.